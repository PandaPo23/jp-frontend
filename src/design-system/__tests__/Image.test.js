import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Image from '../Image';

describe('Image component', () => {
  it('should render image without crashing', () => {
    const wrapper = mount(<Image src="https://source.unsplash.com/random" />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
