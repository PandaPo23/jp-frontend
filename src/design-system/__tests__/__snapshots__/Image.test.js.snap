// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image component should render image without crashing 1`] = `
.emotion-0 {
  display: inline-block;
  height: auto;
  max-width: 100%;
}

<Image
  as="img"
  display="inline-block"
  height="auto"
  maxWidth="100%"
  src="https://source.unsplash.com/random"
>
  <img
    className="emotion-0"
    src="https://source.unsplash.com/random"
  />
</Image>
`;
